docs/quick-start/在苹果笔记本上搭建ByConity开发环境 set up ByConity dev enviroment on mac.md

```diff
@@ -0,0 +1,94 @@
+
+
+# 结论： 
```

Comment:
Can you change this section name to Set up the ByConity development environment? 
BTW,  please also update the markdown file name to set-up-byconity-dev-env-on-mac.mdx

Reply:
I changed as your advice.